### ISSUE TITLE
Status Endpoint Returning Inaccurate State

### ISSUE THREAD

**Issue by alecdoconnor** [tobiasehlert/teslamateapi#92]
**Charging Details**
<img width="463" alt="Screen Shot 2021-07-20 at 2 44 21 PM" src="https://user-images.githubusercontent.com/20458718/126378448-46005625-6852-4879-adf8-96d7dc77e73f.png">

**Driving Details**
<img width="197" alt="Screen Shot 2021-07-20 at 2 47 26 PM" src="https://user-images.githubusercontent.com/20458718/126378786-f66873bc-1638-4459-b71e-c70455ed2c2b.png">

Referencing the images above, the state is showing old data. Today is July 20 and my car is not charging, but it shows the state as "charging" and "state_since" as July 13, about one week ago.

It was charging earlier today and I manually stopped it with the Tesla app, however, it is still plugged in. This may be why it is showing charging. However, I have charged multiple times in the last week.

Additionally, the "driving_details" also shows a -11 kW under power, indicating it is charging at 11 kW. My car was not charging at this speed earlier today, but likely was during the charge one week ago.

The Battery details are also showing old data, as my range/battery_level is at a different level. Curious if anyone else sees these anomalies.

**Comment by alecdoconnor** [comment]
Restarting the container seems to have fixed the issue, I believe there may have just been stale data and MQTT stopped receiving updates. Funny enough, another container was still receiving MQTT messages

**Comment by tobiasehlert** [comment]
Hi @alecdoconnor,

Thanks for reporting, but it seems the data collection on the TeslaMate side has stopped working. TeslaMateApi does not receive any updates from Tesla directly, it collects the data only from Postgres and MQTT and the source of that for those is TeslaMate only.

Regarding "old data" on a page like status, TeslaMateApi subscribes to MQTT and asks for the last value of any topic. Else TeslaMateApi endpoint for status would be completely empty on startup. So this is expected behavior but can be a bit weird when encountering the issue you had.

Could you be more specific in the "restarting the container seems to have fixed the issue", did you mean by restarting TeslaMate related containers or TeslaMateApi?

Regards,
Tobias

**Comment by alecdoconnor** [comment]
I think what happened is
I was running this on my laptop for development. About a week ago I shut down my laptop, it's entirely possible it was during this charging time. After turning it back on, I assume some MQTT connection failed to keep reporting messages. However, I'm not sure why it only affected this MQTT connection. I went ahead and closed it because I can't be sure it's related to TeslaMateAPI. The only reason I suspected was because my other MQTT client was still receiving updated messages
